README.md

```diff
@@ -50,6 +50,12 @@ dotnet publish -r win-x64 -c Release /p:Mode=CoreRT-High
 dotnet publish -r win-x64 -c Release /p:Mode=CoreRT-ReflectionFree
 ```
 
+### To build the 10 kB version of the game
+
+```
+dotnet publish -r win-x64 -c Release /p:Mode=CoreRT-ReflectionFree
```

Comment:
I think this should be `CoreRT-CustomPal` based on what's in the csproj.

I wonder if we should name it `CoreRT-NoRuntime` though - since I call the PAL ("Platform Abstraction Layer") places that abstract the underlying platform (Windows-specific parts are there), not the the runtime.

Reply:
NoRuntime is perfect for me. I do not find good name, so end up with that strange name